src/components/CardListItem.js

```diff
@@ -61,7 +61,7 @@ const stylesWithProps = ({ horizontal, orientation, dimensions }) => {
 
   const width = horizontal
     ? imageWidth() * 0.7
-    : orientation === 'landscape'
+    : orientation === 'landscape' || (orientation === 'portrait' && dimensions.width > 450)
```

Comment:
the photos are cut to the right side , also on landscape mode. it seems that this problem is not generated by the new logic: 
<img width="840" alt="Screenshot 2020-10-22 at 13 27 30" src="https://user-images.githubusercontent.com/43641321/96867214-b1fb6c00-146c-11eb-86db-aac2541023aa.png">